app/src/main/java/de/tum/in/tumcampusapp/component/other/locations/LocationManager.java

```diff
@@ -59,13 +60,18 @@
     };
     private static final StationResult[] DEFAULT_CAMPUS_STATION = {
             new StationResult("Garching-Forschungszentrum", "1000460", Integer.MAX_VALUE),
+            new StationResult("Garching-Technische Universität", "1002070", Integer.MAX_VALUE),
```

Comment:
It seems like this causes an issue because the indices don't line up anymore. For me it shows 'Weihenstephan' although I am at 'Stammgelände' which has index 4 in the array CAMPUS_SHORT. In this array on index 4 ist 'Weihenstephan'

Reply:
:man_facepalming: 
We should probably use a map or something, so we don't need to throw indices around?

Reply:
Not as easy, as you would have to map the relationsships somehow using IDs or something. Feel free to suggest a better way @pfent